README.md

```diff
@@ -24,19 +26,19 @@ First, you will need to set up an account in Stripe. You can easily create a dev
 
 Name: `publishableKey`
 
-The public key can be generated in the dashboard of your Stripe app. Just copy the generated API publishable key to this field.
+The public key can be found in the [dashboard](https://dashboard.stripe.com/apikeys) of your Stripe app. Simply copy the generated API publishable key into this field.
 
 ## Secret Key
 
 Name: `secretKey`
 
-This is the API secret generated at the same time the API key. Just copy the generated API secret to this field.
+The secret key can be found in the [dashboard](https://dashboard.stripe.com/apikeys) of your Stripe app. Simply copy the generated API secret key into this field.
 
 ## Webhook URL
 
 Name: `webhookUrl`
 
-This is the URL you should configure for webhooks in Stripe. If you don't configure the webhooks URL in Stripe you app won't receive the events.
+This is the URL you should configure for webhooks in Stripe. If you don't configure the webhook URL in Stripe, your app won't receive the events.
 
 ## Check Webhooks Signature
```

Comment:
Check resolved comments on this PR: https://github.com/slingr-stack/google-storage-package/pull/1#discussion_r1790138744
And review the Capital case format for titles.